Flutter 3 Error Namespace not specified

I’m using Firebase on Android with:

firebase_core: ^3.13.0  
firebase_messaging: ^15.2.5  
flutter_local_notifications: ^19.0.0`

and everything works fine.  
However, when I add `pusher_beams: ^1.1.2`, I get an error as reported.  
If I remove `pusher_beams: ^1.1.2`, the app works normally again.
----
Flutter 3.27.1 • channel stable
Framework • revision 17025dd882
Tools • Dart 3.6.0 • DevTools 2.40.2
------
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':pusher_beams_android'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

--- a/packages/pusher_beams_android/android/build.gradle
+++ b/packages/pusher_beams_android/android/build.gradle
@@ -25,6 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
+    namespace 'com.pusher.pusher_beams'
     compileSdkVersion 31
 
     compileOptions {

